generator/explores/glean_ping_explore.py

```diff
@@ -26,12 +26,48 @@ def _to_lookml(self, v1_name: Optional[str]) -> List[Dict[str, Any]]:
         }
         # collapse whitespace in the description so the lookml looks a little better
         ping_description = " ".join(ping_descriptions[self.name].split())
-        # insert the description in
-        lookml = super()._to_lookml(v1_name)
-        lookml[0][
-            "description"
-        ] = f"Explore for the {self.name} ping. {ping_description}"
-        return lookml
+
+        views_lookml = self.get_view_lookml(self.views["base_view"])
+
+        # The first view, by convention, is always the base view with the
+        # majority of the dimensions from the top level.
+        base = views_lookml["views"][0]
+        base_name = base["name"]
+
+        joins = []
+        for view in views_lookml["views"][1:]:
+            if view["name"].startswith("suggest__"):
+                continue
+            view_name = view["name"]
+            metric = "__".join(view["name"].split("__")[1:])
+            joins.append(
+                {
+                    "name": view_name,
+                    "relationship": "one_to_many",
+                    "sql": (
+                        f"LEFT JOIN UNNEST(${{{base_name}.{metric}}}) AS {view_name} "
+                        f"ON ${{{base_name}.document_id}} = ${{{view_name}.document_id}}"
```

Comment:
Not every table has a document_id field, so this wouldn't work generically.

Reply:
If I understand correctly, yes. Without a join key, there are issues with counting across different labeled counters.

Reply:
We should maybe drop the primary key completely. See below, I tried to use it and it fails. You should be fine to use the LEFT JOIN without a join key.

<img width="1567" alt="Screen Shot 2021-06-03 at 3 34 33 PM" src="https://user-images.githubusercontent.com/20819040/120701827-5a50c480-c481-11eb-8445-dbde4aebfeea.png">